dockerfiles/ruby23.dockerfile

```
@@ -0,0 +1,37 @@
+ARG FROM
+FROM ${FROM}
```

We can always use  `ruby:2.3`:

```suggestion
FROM ruby:2.3
```

---

Done.